docs/source/formulations_example.rst

```diff
@@ -142,9 +142,7 @@ Make sure to set the bounds wide enough to encompass all anticipated use cases o
 
 .. code-block:: python
 
-    from citrine.resources.parameter_template import ParameterTemplate
-    from citrine.resources.property_template import PropertyTemplate
-    from gemd.entity.bounds import RealBounds
+    from gemd import ParameterTemplate, PropertyTemplate, RealBounds
```

Comment:
Why is this change the recommendation?

Reply:
Because there is no additional functionality made available by importing the citrine-python objects and importing from the gemd-python base package significantly reduces the number of characters.  When someone is just learning the package, they shouldn't need to keep track all this complexity / package structure.

Reply:
I don't like the `gemd` imports for data objects because it breaks the mental model of "the `citrine` wraps the `gemd` lower-level api". My preference in general would be to create a more opinionated top-level package import structure for `citrine`, even if data objects are functional interchangeable between libraries. Happy to defer to your preference here though.

Reply:
That's a good critique.  I'd really like to see citrine-python ergonomics improved generally.  The number of import statements working with citrine-python requires can be really painful.

In this case importing some the GEMD model from citrine-python and some from gemd-python seems worse to me than losing that hierarchical packaging perspective.  But I'll sleep on it.